packages/office-ui-fabric-react/src/components/FocusZone/FocusZone.tsx

```diff
@@ -473,7 +476,8 @@ export class FocusZone extends BaseComponent<IFocusZoneProps, {}> implements IFo
   private _moveFocus(
     isForward: boolean,
     getDistanceFromCenter: (activeRect: ClientRect, targetRect: ClientRect) => number,
-    ev?: Event): boolean {
+    ev?: Event,
+    shouldWrap: boolean = true): boolean {
```

Comment:
it is weird to default this to true. it should default to false.

Reply:
I think the name is misleading, I'll update it to be useDefaultWrap so that it's more clear that when this is true it will do the default wrapping behavior